src/pal/src/include/pal/context.h

```diff
@@ -277,6 +277,57 @@ inline void *FPREG_Xstate_Ymmh(const ucontext_t *uc)
 #define MCREG_Sp(mc)      ((mc).sp)
 #define MCREG_Pc(mc)      ((mc).pc)
 #define MCREG_Cpsr(mc)    ((mc).pstate)
+
+
+#define _ALIGN_UP_16(s) ((s + 15) & ~0xf)
```

Comment:
We already have macros ALIGN_UP and ALIGN_DOWN in pal/src/include/pal/utils.h, so you can just include that header and use them.

Reply:
Revised code no longer needs this.